cmd/keeper/cmd/keeper.go

```diff
@@ -116,11 +116,47 @@ type config struct {
 	pgSUPasswordFile        string
 	pgInitialSUUsername     string
 	pgInitialSUPasswordFile string
+	pgOverrideParameters    []string
 
 	neverMaster             bool
 	neverSynchronousReplica bool
 }
 
+// This function parses the catpured `--postgresql-parameter` flags that are
+// used to override PostgreSQL configurations for a stolon node. The flag
+// `--postgresql-cluster` can be passed multiple times providing the ability to
+// have multipe overrides, thus it is a stringSlice.
+//
+// The usge of the flag is in the following format:
+//
+//  ```
+//  /bin/stolon-keeper --postgresql-parameter key1=value1 \
+//      --postgresql-parameter key2=value2
+//  ```
+//
+// Where key1 and key2 are the postgresql parameters to be overrident, and
```

Comment:
overriden

Reply:
Fixed it along with other spelling mistakes.